GETTING_STARTED.md

```diff
@@ -0,0 +1,875 @@
+
+---
+
+## 📃 Macrometa Document Explained
+
+One of the datastore's that Macrometa supports is **document**, a **document**
+is an object that is a JSON serializable object with the following properties:
+
+- Contains the `_key` field, which identifies the document uniquely within a
+  specific collection.
+- Contains the `_id` field (also called the _handle_), which identifies the
+  document uniquely across all collections within a fabric. This ID is a
+  combination of the collection name and the document key using the format
+  `{collection}/{key}` (see example below).
+- Contains the `_rev` field. GDN supports MVCC (Multiple Version Concurrency
+  Control) and is capable of storing each document in multiple revisions. Latest
+  revision of a document is indicated by this field. The field is populated by
+  GDN and is not required as input unless you want to validate a document
+  against its current revision.
+
+Here is an example of a valid document:
+
+```
+    {
+        '_id': 'students/bruce',
+        '_key': 'bruce',
+        '_rev': '_Wm3dzEi--_',
+        'first_name': 'Bruce',
+        'last_name': 'Wayne',
+        'address': {
+            'street' : '1007 Mountain Dr.',
+            'city': 'Gotham',
+            'state': 'NJ'
+        },
+        'is_rich': True,
+        'friends': ['robin', 'gordon']
+    }
+```
+
+**Edge documents (edges)** are similar to standard documents but with two
+additional required fields `_from` and `_to`. Values of these fields must be the
+handles of "from" and "to" vertex documents linked by the edge document in
+question (see [Graph](docs/Reference/Graph/README.md) for details).
+
+Here is an example of a valid edge document:
+
+```
+    {
+        '_id': 'friends/001',
+        '_key': '001',
+        '_rev': '_Wm3dyle--_',
+        '_from': 'students/john',
+        '_to': 'students/jane',
+        'closeness': 9.5
+    }
+```
+
+---
+
+## 🔐 Auth - Connect to GDN
+
+The first step to start using GDN is establishing a connection to
+[play.macrometa.io](https://play.macrometa.io/). When this code executes, it
+initializes the server connection to your nearest region. You can access your
+Macrometa GDN account using several methods, such as
+[API keys](https://macrometa.com/docs/account-management/api-keys/),
+[User authentication](https://macrometa.com/docs/account-management/auth/user-auth),
+[Token-based authentication (JWT)](https://macrometa.com/docs/account-management/auth/jwts).
+
+### How to create and instance of jsc8?
+
+```js
+const jsc8 = require("jsc8");
+const client = new jsc8("https://gdn1.macrometa.io");
+```
+
+or to have fail-over support:
+
+```js
+const jsc8 = require("jsc8");
+const client = new jsc8([
+  "https://gdn1.macrometa.io",
+  "https://gdn2.macrometa.io",
+]);
+```
+
+This connection string actually represents the default value(
+`"https://gdn1.macrometa.io"` ), so if this is the value you want, you can omit
```

Comment:
If this is the default then we need to change in code and here too to maybe play.paas.macrometa.io